test/test_phaxio.rb

```diff
@@ -69,4 +69,32 @@ def test_get_account_status
     assert_equal "Account status retrieved successfully", @response["message"]
     assert_equal 120, @response["data"]["faxes_sent_this_month"]
   end
+
+  def test_attach_phax_code_to_pdf
+    Phaxio.expects(:send_post).with(
+      '/attachPhaxCodeToPdf',
```

Comment:
While this test will pass, the filename parameter should be used to eventually POST a file to the api, not just the string.  I'm not sure this function actually works to execute a real API call.